packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/angular/remotesignout.md

```diff
@@ -1,4 +1,4 @@
-This navigate the user's browser to the [OIDC logout page](/docs/reference/api/oidc/#logout), and then redirects back to the [postLogoutRedirectUri](https://github.com/okta/okta-auth-js#postlogoutredirecturi) that was specified in the config (or `window.location.origin` if no `postLogoutRedirectUri` was specified). This URI must be one of those listed in the `Logout redirect URI` section of your application's settings. See [Define the signout callback](/docs/guides/sign-users-out/define-signout-callback/).
+This navigate the user's browser to the [OIDC logout page](/docs/reference/api/oidc/#logout), and then redirects back to the [postLogoutRedirectUri](https://github.com/okta/okta-auth-js#postlogoutredirecturi) that was specified in the config (or `window.location.origin` if no `postLogoutRedirectUri` was specified). This URI must be one of those listed in the **Sign-out redirect URIs** section of your application's settings. See [Define the signout callback](/docs/guides/sign-users-out/define-signout-callback/).
```

Comment:
```suggestion
This navigates the user's browser to the [OIDC logout page](/docs/reference/api/oidc/#logout), and then redirects back to the [postLogoutRedirectUri](https://github.com/okta/okta-auth-js#postlogoutredirecturi) that was specified in the config (or `window.location.origin` if no `postLogoutRedirectUri` was specified). This URI must be one of those listed in the **Sign-out redirect URIs** section of your application's settings. See [Define the signout callback](/docs/guides/sign-users-out/define-signout-callback/).
```

Reply:
Changed to "This sends the user's browser..."